ee/migrations/0013_silence_deprecated_tags_warnings.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 3.2.13 on 2022-06-13 17:48
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Checking the SQL for this:

```
$ ./manage.py sqlmigrate ee 0013

🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻
️Environment variable DEBUG is set - PostHog is running in DEVELOPMENT MODE!
Be sure to unset DEBUG if this is supposed to be a PRODUCTION ENVIRONMENT!
🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺


🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻🔻
Skipping async migrations setup. This is unsafe in production!
🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺🔺

AXES: BEGIN LOG
AXES: BEGIN LOG
AXES: Using django-axes version 5.9.0
AXES: Using django-axes version 5.9.0
AXES: blocking by IP only.
AXES: blocking by IP only.
BEGIN;
--
-- Rename field tags on enterpriseeventdefinition to deprecated_tags_v2
--
ALTER TABLE "ee_enterpriseeventdefinition" RENAME COLUMN "tags" TO "deprecated_tags_v2";
--
-- Rename field tags on enterprisepropertydefinition to deprecated_tags_v2
--
ALTER TABLE "ee_enterprisepropertydefinition" RENAME COLUMN "tags" TO "deprecated_tags_v2";
--
-- Alter field deprecated_tags_v2 on enterpriseeventdefinition
--
ALTER TABLE "ee_enterpriseeventdefinition" RENAME COLUMN "deprecated_tags_v2" TO "tags";
--
-- Alter field deprecated_tags_v2 on enterprisepropertydefinition
--
ALTER TABLE "ee_enterprisepropertydefinition" RENAME COLUMN "deprecated_tags_v2" TO "tags";
COMMIT;
```

Bit funny that it renames the column to deprecated_tags_v2, then back to tags, this should be fine though as is in a  transaction.